packages/typespec-client-generator-core/lib/decorators.tsp

```diff
@@ -2,6 +2,19 @@ using TypeSpec.Reflection;
 
 namespace Azure.ClientGenerator.Core;
 
+/**
+ * Changes the name of a method, parameter, property, or model generated in the client SDK
+ * @param rename The rename you want applied to the object
+ * @param scope The language scope you want this decorator to apply to. If not specified, will apply to all language emitters
+ *
+ * @example
+ * ```typespec
+ * @clientName("nameInClient")
+ * op nameInService: void;
+ * ```
+ */
+extern dec clientName(target: unknown, rename: valueof string, scope?: valueof string);
```

Comment:
Have you validated that's the name you want to use?

Reply:
yes we're leaning more towards `clientName`, but will follow up and make sure we're all on the same page